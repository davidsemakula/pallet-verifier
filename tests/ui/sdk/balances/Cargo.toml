[package]
name = "pallet-balances"
version = "28.0.0"
#authors.workspace = true
#edition.workspace = true
edition = "2021"
license = "Apache-2.0"
#homepage.workspace = true
#repository.workspace = true
description = "FRAME pallet to manage balances"
readme = "README.md"

#[lints]
#workspace = true

[package.metadata.docs.rs]
targets = ["x86_64-unknown-linux-gnu"]

[dependencies]
codec = { version = "3.6.12", default-features = false, features = ["derive", "max-encoded-len"], package = "parity-scale-codec" }
log = { version = "0.4.22", default-features = false }
scale-info = { version = "2.11.3", default-features = false, features = ["derive"] }
frame-benchmarking = { version = "38.0.0", optional = true, default-features = false }
frame-support = { version = "38.0.0", default-features = false }
frame-system = { version = "38.0.0", default-features = false }
sp-runtime = { version = "39.0.0", default-features = false }
docify = { version = "0.2.8" }

[dev-dependencies]
pallet-transaction-payment = { version = "38.0.0", default-features = true }
frame-support = { version = "38.0.0", default-features = true, features = ["experimental"] }
sp-core = { version = "34.0.0", default-features = true }
sp-io = { version = "38.0.0", default-features = true }
paste = { version = "1.0.15", default-features = true }

[features]
default = ["std"]
std = [
	"codec/std",
	"frame-benchmarking?/std",
	"frame-support/std",
	"frame-system/std",
	"log/std",
	"pallet-transaction-payment/std",
	"scale-info/std",
	"sp-core/std",
	"sp-io/std",
	"sp-runtime/std",
]
# Enable support for setting the existential deposit to zero.
insecure_zero_ed = []
runtime-benchmarks = [
	"frame-benchmarking/runtime-benchmarks",
	"frame-support/runtime-benchmarks",
	"frame-system/runtime-benchmarks",
	"sp-runtime/runtime-benchmarks",
]
try-runtime = [
	"frame-support/try-runtime",
	"frame-system/try-runtime",
	"pallet-transaction-payment/try-runtime",
	"sp-runtime/try-runtime",
]
