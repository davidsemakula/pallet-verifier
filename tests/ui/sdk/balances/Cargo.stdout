Searching for FRAME pallet definition ...
Searching for dispatchable function definitions ...
Searching for implementations of pallet hooks ...
Searching for public associated function definitions ...
Searching for pallet associated function calls ...
Generating tractable entry points for FRAME pallet ...
Generating specialized summaries ...
Analyzing FRAME pallet with MIRAI ...
Creating summaries for FRAME and Substrate functions ...
Analyzing dispatchables ...
Analyzing dispatchable: `burn` ...
Analyzing dispatchable: `force_adjust_total_issuance` ...
Analyzing dispatchable: `force_set_balance` ...
Analyzing dispatchable: `force_transfer` ...
Analyzing dispatchable: `force_unreserve` ...
Analyzing dispatchable: `transfer_all` ...
Analyzing dispatchable: `transfer_allow_death` ...
Analyzing dispatchable: `transfer_keep_alive` ...
Analyzing dispatchable: `upgrade_accounts` ...
Analyzing hooks ...
Analyzing hook: `<Self as frame_support::traits::Hooks>::integrity_test` ...
Analyzing pub assoc fns ...
Analyzing pub assoc fn: `ensure_upgraded` ...
Analyzing pub assoc fn: `free_balance` ...
Analyzing pub assoc fn: `inactive_issuance` ...
Analyzing pub assoc fn: `locks` ...
Analyzing pub assoc fn: `reserved_balance` ...
Analyzing pub assoc fn: `reserves` ...
Analyzing pub assoc fn: `total_issuance` ...
Analyzing pub assoc fn: `usable_balance` ...
Analyzing pub assoc fn: `usable_balance_for_fees` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::Currency>::active_issuance` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::Currency>::burn` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::Currency>::can_slash` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::Currency>::deactivate` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::Currency>::deposit_creating` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::Currency>::deposit_into_existing` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::Currency>::ensure_can_withdraw` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::Currency>::free_balance` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::Currency>::issue` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::Currency>::make_free_balance_be` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::Currency>::minimum_balance` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::Currency>::reactivate` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::Currency>::slash` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::Currency>::total_balance` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::Currency>::total_issuance` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::Currency>::transfer` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::Currency>::withdraw` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::InspectLockableCurrency>::balance_locked` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::LockableCurrency>::extend_lock` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::LockableCurrency>::remove_lock` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::LockableCurrency>::set_lock` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::NamedReservableCurrency>::repatriate_reserved_named` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::NamedReservableCurrency>::reserve_named` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::NamedReservableCurrency>::reserved_balance_named` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::NamedReservableCurrency>::slash_reserved_named` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::NamedReservableCurrency>::unreserve_named` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::ReservableCurrency>::can_reserve` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::ReservableCurrency>::repatriate_reserved` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::ReservableCurrency>::reserve` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::ReservableCurrency>::reserved_balance` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::ReservableCurrency>::slash_reserved` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::ReservableCurrency>::unreserve` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::fungible::Balanced>::done_deposit` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::fungible::Balanced>::done_issue` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::fungible::Balanced>::done_rescind` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::fungible::Balanced>::done_withdraw` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::fungible::Inspect>::active_issuance` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::fungible::Inspect>::balance` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::fungible::Inspect>::can_deposit` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::fungible::Inspect>::can_withdraw` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::fungible::Inspect>::minimum_balance` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::fungible::Inspect>::reducible_balance` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::fungible::Inspect>::total_balance` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::fungible::Inspect>::total_issuance` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::fungible::InspectFreeze>::balance_frozen` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::fungible::InspectFreeze>::can_freeze` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::fungible::InspectHold>::balance_on_hold` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::fungible::InspectHold>::hold_available` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::fungible::InspectHold>::reducible_total_balance_on_hold` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::fungible::InspectHold>::total_balance_on_hold` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::fungible::Mutate>::done_burn_from` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::fungible::Mutate>::done_mint_into` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::fungible::Mutate>::done_restore` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::fungible::Mutate>::done_shelve` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::fungible::Mutate>::done_transfer` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::fungible::MutateFreeze>::extend_freeze` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::fungible::MutateFreeze>::set_freeze` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::fungible::MutateFreeze>::thaw` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::fungible::Unbalanced>::deactivate` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::fungible::Unbalanced>::handle_dust` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::fungible::Unbalanced>::reactivate` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::fungible::Unbalanced>::set_total_issuance` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::fungible::Unbalanced>::write_balance` ...
Analyzing pub assoc fn: `<Self as frame_support::traits::fungible::UnbalancedHold>::set_balance_on_hold` ...
